frontend/src/libs/components/customer-order-card/libs/helpers/get-badge-color-by-status.helper.ts

```diff
@@ -0,0 +1,23 @@
+import { OrderStatus } from '~/packages/orders/orders.js';
+
+import { ProgressStatus } from '../enums/progress-status.js';
+import { type BadgeStatus } from '../types/badge-status.type.js';
+
+const getBadgeColorByStatus = (status: string): BadgeStatus => {
```

Comment:
let's make a map for getting needed info
example - mapOrderStatusToReadable in fe